### DISCUSSION TITLE
ZY-M100

### DISCUSSION THREAD

**Discussion by cmussoni** [indomus/forum#2569] [HUB: Home Assistant]
Ciao a tutti,
ho acquistato 2 ZY-M100, versione ZigBee.
In HomeAssistant tramite ZHA li rilevo immediatamente. Il problema e' che non si vedono assolutamente nessuna delle entita'.
Sapete per caso se e' possibile fare qualcosa in merito ?
Su Indomus avevo letto che erano compatibili con ZHA.
Grazie
Claudio

**Comment by cmussoni** [comment]
Il modello riportato su ZHA e': TS0601 by _TZE204_sxm7l9xa

**Comment by indomus** [comment]
Crea un cartella in /config chiamata “custom_zha_quirks”.
Poi in configuration.yaml aggiungi questa riga:

```
zha:
  custom_quirks_path: /config/custom_zha_quirks/
```

Poi nella cartella creata in precedenza (“custom_zha_quirks”) carica il contenuto (decompattato) di questo file :
https://github.com/zigpy/zha-device-handlers/files/11540754/ts0601_motion.zip

riavvia e verifica dovresti avere i tuoi sensori

**Reply by cmussoni** [reply]
Ciao e grazie per la risposta.
Ho caricato il file zip ma ancora niente. Ho provato poi a caricare direttamente il file "ts0601_motion.py" contenuto nel file zip e finalmente ho ottenuto la mia prima entita' "Occupancy". A questo punto nel link github ho notato che c'era il file "ts0601_illuminance.py", ho caricato anche questo nella stessa cartella ed ho ottenuto la seconda entita' "Illuminance". Ora mi aspetto di trovare (forse) anche l'entita' "distanza" ed i vari settaggi per impostare i vari parametri che si possono modificare (ritardi in tempo, distanze e sensibilita'). Non so se sto sbagliando qualcosa io, ma e' l'unico modo che mi ha permesso di usare questo sensore che comunque reagisce molto velocemente ed in modo alla presenza. Claudio.

**Reply by KC-inDomus** [reply]
no Claudio l'entità distanza ce l'hai solo (per ora) con zigbee2mqtt

**Comment by dany85free** [comment]
Ho scaricato il file zip ma il file ts601_illuminance.py non c'è; potete aggiornare il file zip oppure condividerlo? Grazie
